### ISSUE TITLE
running tailscale up on an expired node doesn't reauthenticate the node

### ISSUE THREAD

**Issue by dannypagano** [tailscale/tailscale#6786]
### What is the issue?

The authentication key for a node expired and I was unable to reauthenticate by following the [instructions in the docs](https://tailscale.com/kb/1028/key-expiry/). 

Running `sudo tailscale up` over tailscale ssh and normal ssh returned an empty new line, rather than the authentication login url.

running `sudo tailscale down` to logout, followed by `sudo tailscale up` worked, and generated the auth login url as expected.

### Steps to reproduce

My node's auth key expired, so I connected over Tailscale SSH and ran `sudo tailscale up`. 

### Are there any recent changes that introduced the issue?

recently updated the installed version of tailscale, but unknown whether this caused the issue since it was updated before the key expired.

### OS

Synology

### OS version

DSM 7.1.1-42962 Update 2

### Tailscale version

1.30.2

### Bug report

BUG-20bc95fc7c98800de1cceab2dacb8a3624d74acbb816514901c84517207b2479-20221219194955Z-012a1e63626487c2

**Comment by kennyparsons** [comment]
same issue on debian 11
1.34.1
  tailscale commit: 328b49c4d69bd7fe1cbc8474ee36854dc52eb22c
  other commit: 921b59a2ebc3eb4a703d93f504fc9e10fddf64f6
  go version: go1.19.2-ts3fd24dee31

**Comment by sellout** [comment]
This is perhaps what’s affecting me as well.

I had three nodes expire today. I followed https://tailscale.com/kb/1028/key-expiry/

I extended the key on the first one, but then couldn’t get `tailscale up --ssh` to re-auth.

Once my local machine expired a bit later, `tailscale up` _did_ re-auth, but on the two remote machines, I get disconnected as soon as it expires, so I haven’t been able to get `tailscale up` to work.

I later discovered `--force-reauth`, but `tailscale up --ssh --force-reauth` on the remote machines just hangs, so I don’t know what to do there.

The taliscale versions  range from 1.32.3 to 1.36.1.
